infra/aws/terraform/prow-build-cluster/resources/test-pods/externalsecrets.yaml

```diff
@@ -4,78 +4,114 @@
-#     name: ssh-public                            # The key to write to in the Kubernetes Secret
-#     version: latest                             # The version of the GSM Secret
-#   - key: prow-build-ssh-key-secret-ssh-private  # The name of the GSM Secret
-#     name: ssh-private                           # The key to write to in the Kubernetes Secret
-#     version: latest                             # The version of the GSM Secret
-# ---
-# apiVersion: kubernetes-client.io/v1
-# kind: ExternalSecret
-# metadata:
-#   name: aws-credentials-768319786644           # The name of the Kubernetes Secret
-#   namespace: test-pods
-# spec:
-#   backendType: gcpSecretsManager
-#   projectId: k8s-infra-prow-build
-#   data:
-#   - key: gke_k8s-prow-builds_us-central1-f_prow__test-pods__aws-credentials-768319786644   # The name of the GSM Secret
-#     name: credentials                                                                      # The key to write to in the Kubernetes Secret
-#     version: latest                                                                        # The version of the GSM Secret
-#     property: credentials
-# ---
-# apiVersion: kubernetes-client.io/v1
-# kind: ExternalSecret
-# metadata:
-#   name: aws-credentials-607362164682           # The name of the Kubernetes Secret
-#   namespace: test-pods
-# spec:
-#   backendType: gcpSecretsManager
-#   projectId: k8s-infra-prow-build
-#   data:
-#   - key: gke_k8s-prow-builds_us-central1-f_prow__test-pods__aws-credentials-607362164682   # The name of the GSM Secret
-#     name: credentials                                                                      # The key to write to in the Kubernetes Secret
-#     version: latest                                                                        # The version of the GSM Secret
-#     property: credentials
-# ---
-# apiVersion: kubernetes-client.io/v1
-# kind: ExternalSecret
-# metadata:
-#   name: aws-ssh-key-secret           # The name of the Kubernetes Secret
-#   namespace: test-pods
-# spec:
-#   backendType: gcpSecretsManager
-#   projectId: k8s-infra-prow-build
-#   data:
-#   - key: gke_k8s-prow-builds_us-central1-f_prow__test-pods__aws-ssh-key-secret   # The name of the GSM Secret
-#     name: aws-ssh-private                                                        # The key to write to in the Kubernetes Secret
-#     version: latest                                                              # The version of the GSM Secret
-#     property: aws-ssh-private
-#   - key: gke_k8s-prow-builds_us-central1-f_prow__test-pods__aws-ssh-key-secret   # The name of the GSM Secret
-#     name: aws-ssh-public                                                         # The key to write to in the Kubernetes Secret
-#     version: latest                                                              # The version of the GSM Secret
-#     property: aws-ssh-public
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ClusterSecretStore
```

Comment:
We only want to sync secrets in a specific namespace. we should scope this to the namespace.